hapi-fhir-validation/src/main/java/org/hl7/fhir/common/hapi/validation/validator/ValidatorWrapper.java

```diff
@@ -236,6 +236,23 @@ public List<ValidationMessage> validate(
 		return messages;
 	}
 
+	private InstanceValidator getInstanceValidator(IWorkerContext theWorkerContext) {
```

Comment:
```suggestion
	private InstanceValidator buildInstanceValidator(IWorkerContext theWorkerContext) {
```

suggestion(non-blocking): IMO getInstanceValidator implies returning an existing validator object. build implies a fresh one on every call.